tests/phpunit/tests/theme/themeDir.php

```diff
@@ -45,16 +45,16 @@ function _theme_root($dir) {
 	 */
 	function test_theme_default() {
 		$themes = get_themes();
-		$theme = get_theme('WordPress Default');
-		$this->assertEquals( $themes['WordPress Default'], $theme );
+		$theme = get_theme('ClassicPress Default');
+		$this->assertEquals( $themes['ClassicPress Default'], $theme );
 
 		$this->assertFalse( empty($theme) );
 
 		#echo gen_tests_array('theme', $theme);
 
-		$this->assertEquals( 'WordPress Default', $theme['Name'] );
-		$this->assertEquals( 'WordPress Default', $theme['Title'] );
-		$this->assertEquals( 'The default WordPress theme based on the famous <a href="http://binarybonsai.com/kubrick/">Kubrick</a>.', $theme['Description'] );
+		$this->assertEquals( 'ClassicPress Default', $theme['Name'] );
```

Comment:
There is no such theme, maybe keep this as WordPress default theme or maybe reference it as the Kubrick theme instead?
This comment is for L55-57

Reply:
In this case (test theme data), `ClassicPress Default` is fine.  This value is read from the file `tests/phpunit/data/themedir1/default/style.css` which has also been modified as part of this PR.